pkg/upgrade/upgrade.go

```diff
@@ -114,7 +114,7 @@ func OperatorUninstall(cli client.Client, cfg *rest.Config) error {
 	} else if platform == deploy.ManagedRhods {
 		subsName = "addon-managed-odh"
 	}
-	sub, _ := deploy.SubscriptionExists(cli, operatorNs, subsName)
+	sub, _ := deploy.GetSubscription(cli, operatorNs, subsName)
```

Comment:
I wonder why are we ignoring the error here. Shouldn't we at least log it or ignore if `apierrs.IsNotFound(err)` is what's returned? This would imply changes in the ` GetSubscription` but maybe we should just return any error there and let the caller decide how to handle it?

Reply:
did some updates, also cleanup the IsNotFound check